### PR TITLE
Feature/workspaces

### PR DESCRIPTION
**Pull request by martinbreu**
Dieser PR dient erstmal zur einfacheren Zusammenarbeit.

### CODE REVIEW

**Review by alexanderkiel — COMMENTED** [review]
Looks good, but we have to test it. So I'll ask ML to set the Domains, so Let's Encrypt can see us.

**Comment by martinbreu** [comment]
`terraform plan` looks fine, should we try `terraform apply` together at https://whereby.com/biocoders ?

**Comment by martinbreu** [comment]
`terraform apply`:
```
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [5m20s elapsed]
module.postgres.opentelekomcloud_rds_instance_v3.postgres: Creation complete after 5m22s [id=ecfbce3524c94d2489ad8dfbf6b3cbf6in03]
module.postgres.opentelekomcloud_dns_recordset_v2.postgres: Creating...
module.postgres.opentelekomcloud_dns_recordset_v2.postgres: Creation complete after 6s [id=ff8080826c8f5cf2016cdc5fe93c4208/ff8080826c8f5c5c016cdc64e3d54f38]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [5m30s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [5m40s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [5m50s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [6m0s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [6m10s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [6m20s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [6m30s elapsed]
module.server.opentelekomcloud_compute_instance_v2.server: Still creating... [6m40s elapsed]


Error: timeout - last error: dial tcp :22: connect: connection refused


martinbreu@nful050x:~/Repositories/itc/open-telekom-cloud$ 
```
Server wurde erstellt, keine EIP zugewiesen
Nochmaliges apply, gleiches Ergebnis.

**Comment by martinbreu** [comment]
```
module.postgres.opentelekomcloud_rds_instance_v3.postgres: Still creating... [5m10s elapsed]
module.postgres.opentelekomcloud_rds_instance_v3.postgres: Creation complete after 5m19s [id=2cfc2ca37fcd4a2e92041653169cee38in03]
module.postgres.opentelekomcloud_dns_recordset_v2.postgres: Creating...
module.postgres.opentelekomcloud_dns_recordset_v2.postgres: Creation complete after 6s [id=ff8080826c8f5c5c016cdd015f67543f/ff8080826c8f5c5c016cdd065d54546a]

Error: Upload failed: scp: /mnt/secrets/searchbroker-ui: Permission denied


martinbreu@nful050x:~/Repositories/itc/open-telekom-cloud$ 
```

**Comment by martinbreu** [comment]
Anyhow, we need to switch at least the database between dev and prod freely. I would suggest a manuel way

**Comment by alexanderkiel** [comment]
Still work in progress

File: modules/server/main.tf [hunk 1]
```diff
@@ -22,8 +22,8 @@ module "certbot" {
 
 module "searchbroker" {
   source                  = "./searchbroker"
-  searchbroker_version    = "2.2.0"
-  searchbroker-ui_version = "6e41bbd9c8c77713d053c1b7a6d6f3d9b06081d3"
+  searchbroker_version    = "3.3.0-SNAPSHOT"
```
Comment (alexanderkiel) [inline modules/server/main.tf]: SNAPSHOT should go before finalizing this

**Review by alexanderkiel — CHANGES_REQUESTED** [review]

**Comment by martinbreu** [comment]
All services run as expected

**Review by alexanderkiel — APPROVED** [review]

**Pull request merged**